_posts/en/newsletters/2020-11-25-newsletter.md

```diff
@@ -0,0 +1,146 @@
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [What is fuzz testing?]({{bse}}99955)
+  Michael Folkson summarizes [fuzz testing][fuzzing wikipedia], a technique of feeding a program like
+  Bitcoin Core varying malformed inputs for purposes of finding bugs.
+
+- [Adding instead of concatenating hashes in Merkle trees]({{bse}}100098)
+  Pieter Wuille breaks down a variety of issues to consider regarding the idea
+  of combining hashes in a merkle tree using an operation such as
+  addition instead of the concatenation that Bitcoin currently uses.
+
+- [How will the migration tool from a Bitcoin Core legacy wallet to a descriptor wallet work?]({{bse}}99624)
+  Michael Folkson outlines how a tool that migrates existing legacy wallets to
+  [descriptor][topic descriptors] wallets might work.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [C-Lightning 0.9.2][C-Lightning 0.9.2] is the release for
+  the next maintenance version of C-Lightning.  It contains "new CLI-level
+  notifications, better channel state reporting, and stable plugin-hook
+  call ordering" in addition to other new features and bug fixes.  The
+  release notes do warn that [PSBTs][topic psbt] generated by this
+  version of C-Lightning will be rejected by some older versions of
+  Bitcoin Core (versions 0.20.0 and earlier); this is a consequence of a
+  change made to the [PSBT specification][news109 bips948] and [Bitcoin
+  Core][news105 core19215] to defend against a possible [fee overpayment
+  attack][].
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+*Note: some commits to Bitcoin Core mentioned below apply to its
+development branch and so those changes will likely not be released
+until version 0.22, about six months after the release of the upcoming
+version 0.21.*
+
+- [Bitcoin Core #20305][] starts shifting RPC fee rate units to satoshis
+  per vbyte (sat/vB) rather than BTC per 1,000 vbytes (BTC/kvB) by introducing a
+  `fee_rate` parameter to the `sendtoaddress`,
+  `sendmany`, `fundrawtransaction`, and `walletcreatefundedpsbt` RPCs as well as
+  to the experimental [new][news116 send] `send` RPC. In addition, the `bumpfee` RPC `fee_rate`
+  option is changed from BTC/kB to sat/vB. Users are warned that the latter is a
+  breaking API change, but it should be relatively [benign][feerate change
+  benign]. The legacy `feeRate` option in RPCs `fundrawtransaction` and
+  `walletcreatefundedpsbt` still exists for setting a fee rate in BTC/kvB; it is
```

Comment:
s/fee rate/feerate/

Latter preferred according to style guide, and also appears that way a couple times in this paragraph.

Reply:
I've been writing "fee rate" lately as that's what has been used recently in bitcoin core documentation and param naming and what people seem to prefer (e.g. I initially planned to add a feerate param/option, as it's shorter, but reviewers asked for fee_rate).

Reply:
I agree with @Xekyo here. The style guide says 'feerate' not 'fee rate'.

(I'm happy to discuss elsewhere whether we should update the style guide, but we should stick to it in our newsletters for consistency)

Reply:
STGM -- sorry, I forgot to check the guide on this 😇